unexpand/src/tab_stops.rs

```diff
@@ -0,0 +1,183 @@
+#[derive(Debug)]
+pub struct TabStops {
+    offset:    Option<usize>,
+    repetable: Option<usize>,
+    positions: Vec<usize>,
+}
+
+impl TabStops {
+    pub fn new(tabs_str: Option<&str>) -> TabStops {
+        match tabs_str {
+            Some(tabs_str) => {
+                if tabs_str == "" {
+                    return TabStops { offset: None, repetable: Some(8), positions: vec![] };
+                }
+
+                let tabs_str = tabs_str.replace(", ", ",");
+                let mut tabs_vec: Vec<&str> =
+                    tabs_str.split(|c| c == ',' || c == ' ').map(|s| s.trim()).collect();
+
+                if tabs_vec.len() == 1 {
+                    let value = tabs_vec[0].parse::<usize>().unwrap();
```

Comment:
There is a awful lot of unwrapping in the code, there is any guarantee that these uses of unwrap won't panic?